test/trajectory-subsetter/TrajectorySubsetter_Regression.ipynb

```diff
@@ -402,7 +407,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.12"
+   "version": "3.11.5"
   },
   "name": "SDS_Regression.ipynb"
```

Comment:
I feel like I've asked this before, but why does this metadata list "SDS_Regression.ipynb" and not "TrajectorySubsetter_Regression.ipynb"?

Reply:
That's probably very very old.  It can be cleaned up, but doesn't need to be.